script/openqa-bootstrap

```diff
@@ -16,15 +19,15 @@ zypper -n install --no-recommends openQA-local-db apache2 openQA-worker qemu-kvm
 
 # setup database
 systemctl enable --now postgresql
-su postgres -c "createuser -D geekotest"
-su postgres -c "createdb -O geekotest openqa"
+getent passwd $dbuser || su postgres -c "createuser -D $dbuser"
```

Comment:
So if the *UNIX* user already exists it doesn't create the *PostgreSQL* user? Might work in practice but seems messy too me.